src/generic/infobar.cpp

```diff
@@ -115,19 +117,37 @@ bool wxInfoBarGeneric::Create(wxWindow *parent, wxWindowID winid)
     if(!m_hasFgCol)
         m_text->SetForegroundColour(colFg);
 
-    m_button = wxBitmapButton::NewCloseButton(this, wxID_ANY);
+    m_button = wxBitmapButton::NewCloseButton(this, wxID_CLOSE);
     m_button->SetToolTip(_("Hide this notification message."));
 
+    m_dontShowAgainCheckbox =
+        new wxCheckBox(this, wxID_ANY, _(L"Do not show this again."), wxDefaultPosition,
```

Comment:
Why `L`?
```suggestion
        new wxCheckBox(this, wxID_ANY, _("Do not show this again."), wxDefaultPosition,
```

Reply:
Sorry, force of habit. I use `L` in my code usually, but not in wx PRs.